cmd/chaininfo/chain_info.go

```diff
@@ -0,0 +1,159 @@
+// Copyright 2021-2022, Offchain Labs, Inc.
+// For license information, see https://github.com/nitro/blob/master/LICENSE
+
+package chaininfo
+
+import (
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	"math/big"
+	"os"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/params"
+)
+
+//go:embed arbitrum_chain_info.json
+var DefaultChainInfo []byte
+
+type ChainInfo struct {
+	ChainName             string                 `json:"chain-name"`
+	ParentChainId         uint64                 `json:"parent-chain-id"`
+	ChainParameters       *json.RawMessage       `json:"chain-parameters"`
+	ChainConfig           *params.ChainConfig    `json:"chain-config"`
+	RollupAddressesConfig *RollupAddressesConfig `json:"rollup"`
+}
+
+type RollupAddressesConfig struct {
+	Bridge                 string `koanf:"bridge"`
+	Inbox                  string `koanf:"inbox"`
+	SequencerInbox         string `koanf:"sequencer-inbox"`
+	Rollup                 string `koanf:"rollup"`
+	ValidatorUtils         string `koanf:"validator-utils"`
+	ValidatorWalletCreator string `koanf:"validator-wallet-creator"`
+	DeployedAt             uint64 `koanf:"deployed-at"`
```

Comment:
Actually, shouldn't ChainInfo just contain a `RollupAddresses` directly since it implements JSON deserialization? Then I think we could get rid of `RollupAddressesConfig`.

Reply:
This was missed while copying RollupAddressesConfig from arbnode/node.go, fixed now.